File: problems/hard/password_Leak/_tests_/password_Leak.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

#include "../../hard/password_Leak/password_leak.c"

void test(){
    if (passwords("cbaebabacd","abc") != {0, 6})
    {
        printf("✅ c test passed")
    }else{
        printf("❌ c test failed")
    }
    
}

int main(){
    test();

    return 0;
}

File: problems/hard/password_Leak/_tests_/password_Leak.py
import sys
import os

sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))

from password_Leak import  passwordLeaked

def password_leakTest():
    if passwordLeaked("cbaebabacd", "abc") != [0,6]:
        print("❌ failed python test")
    else:
        print("✅ passed python tests")

